WordPress/Sniffs/NamingConventions/PrefixAllGlobalsSniff.php

```diff
@@ -58,13 +58,13 @@ final class PrefixAllGlobalsSniff extends AbstractFunctionParameterSniff {
 	/**
 	 * Minimal number of characters the prefix needs in order to be valid.
 	 *
-	 * @since 2.2.0
+	 * @since n.e.x.t.
```

Comment:
This is not a correct syntax.

The `@since 2.2.0` tag should be left as is, and the next version where this update is going to be released has to be added under it with the explanation of what was changed, possibly link to the plugin handbook where this change is mentioned.

Reply:
ok! I've reverted.